### PR TITLE
Remove azure broker from kyma installation

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

**_Blocked until enabling Azure Broker as the bundle_**

Changes proposed in this pull request:

- Removed the azure broker from kyma installation

**Related issue(s)**

See the PRs in the commercialization and cont-integration repo.

### CODE REVIEW

File: docs/kyma/docs/034-inst-install-subcomponents.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ To specify whether to install a given core subcomponent on Minikube, use the `ma
 
 Example:
 
-To enable the Azure Broker subcomponent, run the following command:
+To disable the API-Controller subcomponent, run the following command:
```
Comment (polskikiel) [inline docs/kyma/docs/034-inst-install-subcomponents.md]: todo: test this

Comment (piotrmiskiewicz) [reply docs/kyma/docs/034-inst-install-subcomponents.md]: why do you describe PAI controller in this PR?

Comment (polskikiel) [reply docs/kyma/docs/034-inst-install-subcomponents.md]: discussed. it was used as the example

**Comment by piotrmiskiewicz** [comment]
please remove resources/core/charts/azure-broker folder

**Pull request merged**